### DISCUSSION TITLE
VictoryAxis - tickLabelComponent

### DISCUSSION THREAD

**Discussion by samynaj** [FormidableLabs/victory#2595] [Q&A]
Please anyone tried to give a custom component to tickLabelComponent in react native, it only takes the VictoryLabel any custom react native element doesn't display anything. Plus I want to be able to handle click events on the component. Urgent suggestions needed here pls.

**Comment by damithg-dev** [comment]
@samynaj have you fond any solution?

**Reply by samynaj** [reply]
Not yet, I had to tell my company design team that the chart pattern in the requirement was not obtainable from the library.

**Comment by nambiarn** [comment]
@samynaj what specific actions are you looking to do with `tickLabelComponent`? outside of using `VictoryLabel`, you can use other tags through react-native-svg to perform actions beyond `VictoryLabel`. Please share your specific use case, and we can help you.
